canonicalwebteam/discourse/parsers/docs.py

```diff
@@ -744,3 +746,20 @@ def _generate_headings_map(self, soup):
                 previous_tag = current_tag
 
         return headings_map
+
+    def _disable_toc(self, soup):
+        disable_toc = False
+
+        first_table = soup.select_one("table:nth-of-type(1)")
+
+        if first_table and (
+            first_table.findAll("th")[0].getText() == "Key"
+            and first_table.findAll("th")[1].getText() == "Value"
+        ):
+            if first_table.findAll("td")[0].text == "disable_toc":
```

Comment:
You think `disable_table_of_contents` is too long?

Reply:
I am happy to use `disable_table_of_contents`